theano/sandbox/cuda/opt.py

```diff
@@ -370,7 +370,7 @@ def local_gpu_split(node):
         if (input.owner and isinstance(input.owner.op, HostFromGpu) or
             any(c != 'output' and isinstance(c.op, GpuFromHost) for c, idx
                 in outs_clients)):
-            new_op = GpuSplit(node.op.len_splits)
+            new_op = GpuSplit(node.op._props_dict()['len_splits'])
```

Comment:
Here the ticket was asking that this was done:
GpuSplit(*node.op._props_dict())

This way, if a new props is done on the CPU, it will make the optimization crash if we don't update the GPU op. I'll kill travis as it is kind of busy and won't be useful for this commit now.